src/components/robot-item-form.tsx

```diff
@@ -0,0 +1,157 @@
+import { makeStyles, TextField, Button } from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import React, { useState, useEffect } from 'react';
+import { successMsg } from '../util/alerts';
+import fakePlaces from '../mock/data/places';
+
+interface robotLoopFormProps {
+  fleetName: string;
+  requestLoop(
+    fleetName: string,
+    numLoops: number,
+    startLocationPoint: string,
+    endLocationPoint: string,
+  ): void;
+}
+
+export const RobotLoopForm = (props: robotLoopFormProps) => {
+  const { requestLoop, fleetName } = props;
+  const classes = useStyles();
+
+  const [listOfPlacesToGo, setListOfPlacesToGo] = useState(['']);
+
+  const [numLoops, setNumLoops] = useState(0);
+  const [startLocation, setStartLocation] = useState('');
+  const [finishLocation, setFinishLocation] = useState('');
+  // Error states
+  const [numLoopsError, setNumLoopsError] = useState('');
+  const [startLocationError, setStartLocationError] = useState('');
+  const [finishLocationError, setFinishLocationError] = useState('');
+
+  useEffect(() => {
+    const listOfPlaces = fakePlaces()[fleetName];
+    setListOfPlacesToGo(listOfPlaces);
+    !listOfPlacesToGo && console.error('List of places to go it`s empty');
+    setStartLocation(listOfPlaces[0]);
+    setFinishLocation(listOfPlaces[1]);
```

Comment:
We should check if `listOfPlaces` has length >= 2

Reply:
You're totally right! Thanks!